src/pages/HomePage/HomePage.js

```diff
@@ -1,19 +1,19 @@
 import React from 'react';
 import './HomePage.scss';
 
-import { homeImgURL } from '../../assets/resources/imagesURL';
+import { arrIMG } from '../../assets/resources/imagesURL';
+
+const randomNum = (Math.random() * 9).toFixed();
```

Comment:
`.toFixed()` returns a string
so at line 14
you access an item from the array, like
```js
arrIMG["1"]
arrIMG["2"]
```
it works, but you can also pass a number.

```suggestion
const randomNum = Math.floor(Math.random() * 9);
```